Sources/Basic/TemporaryFile.swift

```diff
@@ -0,0 +1,172 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright 2016 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+
+ -------------------------------------------------------------------------
+
+ This file provides API to create and use temporary files and directories.
```

Comment:
Off-topic: I personally skip this section for files where the real documentation is attached to actual types or classes. Otherwise it just grows stale and is an extra thing to maintain.